train/src/config.py

```diff
@@ -8,7 +8,10 @@ def migrate(config: dict):
         config["num_best_models_to_keep"] = config["num_models_to_keep"]
         del config["num_models_to_keep"]
 
-    return config
+    return {
+        "lr": 0.9,
+        **config,
+    }
```

Comment:
将来lrの値を別の値に代入するみたいになったとき、lrが値としてほしくなりそう。
意外とこうしといたほうが良いかもしれない･･･？

```suggestion
    if "lr" not in conifg:
        config["lr"] = 0.9

    return config
```